src/dali_executor/CMakeLists.txt

```diff
@@ -28,11 +28,17 @@ set(
 
 include(${tritondalibackend_SOURCE_DIR}/cmake/dali.cmake)
 
+if (${DALI_DOWNLOAD_NIGHTLY})
+    set(DALI_DOWNLOAD_PACKAGE_NAME nvidia-dali-nightly-cuda110$<$<BOOL:${DALI_VERSION}>:==${DALI_VERSION}>)
```

Comment:
Can you infer the CUDA version available in this image somehow (I'm just forward looking)?

Reply:
done